python/dlr/tf_model.py

```diff
@@ -0,0 +1,153 @@
+# coding: utf-8
+import os
+import tensorflow as tf
+
+# A prefix that will be prepended to the names in graph_def
+PREFIX = "import"
+
+
+class TFModelImpl:
```

Comment:
Are you planning to implement the proposed RFC #46 later?

Reply:
`DLRModel` refactoring is a separate activity. It should not change `TFModelImpl`.
It will be another PR to refactor `DLRModel` as proposed in #46.  `DLRModel` will use `TFModelImpl` if model folder contains single `.pb` file